Python/Desktop Time Tracker/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Time Tracker
```

Comment:
Can you name the folder `Desktop-Time-Tracker` instead of `Desktop Time Tracker`?

Reply:
Changed to Desktop_Time_Tracker to match other formats, Done